kythe/rust/indexer/src/error.rs

```diff
@@ -38,10 +38,5 @@ quick_error! {
         }
         // The KytheWriter encounters an error
         WriterError {}
-        // An unknown error occured
```

Comment:
Would be nice to say in the commit description why this is removed.  It's not obvious.

Reply:
Ah, my PR message had a typo. I have fixed that and also provided more in-depth reasoning about why the removal was necessary